src/components/task-plan/homework/exercises.cjsx

```diff
@@ -0,0 +1,213 @@
+    if confirm('Are you sure you want to remove this exercise?')
+      TaskPlanActions.removeExercise(@props.planId, @props.exercise)
+
+  renderHeader: ->
+    unless @props.index is 0
+      moveUp =
+        <BS.Button onClick={@moveExerciseUp} className="btn-xs -move-exercise-up">
+          <i className="fa fa-arrow-up"/>
+        </BS.Button>
+
+    # TODO: Add conditional logic for displaying this button
+    moveDown =
+      <BS.Button onClick={@moveExerciseDown} className="btn-xs -move-exercise-down">
+        <i className="fa fa-arrow-down"/>
+      </BS.Button>
+
+    <span className="-exercise-header">
+      <span className="-exercise-number">{@props.index + 1}</span>
+      <span className="pull-right card-actions">
+        {moveUp}
+        {moveDown}
+        <BS.Button onClick={@removeExercise} className="btn-xs -remove-exercise">
+          <i className="fa fa-close"/>
+        </BS.Button>
+      </span>
+    </span>
+
+  getPanelStyle: ->
+    "default"
+
+  render: ->
+    @renderExercise()
+
+AddExerciseCard = React.createClass
+  displayName: 'AddExerciseCard'
+
+  propTypes:
+    planId: React.PropTypes.any.isRequired
+    exercise: React.PropTypes.object.isRequired
+
+  mixins: [ExerciseCardMixin]
+
+  toggleExercise: ->
+    if TaskPlanStore.hasExercise(@props.planId, @props.exercise.id)
+      TaskPlanActions.removeExercise(@props.planId, @props.exercise)
+    else
+      TaskPlanActions.addExercise(@props.planId, @props.exercise)
+
+  renderHeader: ->
+    active = TaskPlanStore.hasExercise(@props.planId, @props.exercise.id)
+    toggleText = if not active then <span>+</span> else <span>-</span>
+    <BS.Button bsStyle="primary" onClick={@toggleExercise} className="-add-exercise">{toggleText}</BS.Button>
+
+  getPanelStyle: ->
+    if TaskPlanStore.hasExercise(@props.planId, @props.exercise.id)
+      return "info"
+    else
+      return "default"
+
+  render: ->
+    @renderExercise()
+
+ExercisesRenderMixin =
+  componentWillMount:   -> ExerciseStore.addChangeListener(@update)
+  componentWillUnmount: -> ExerciseStore.removeChangeListener(@update)
+
+  update: ->
+    @setState({})
+
+  renderLoading: ->
+    {courseId, pageIds} = @props
+
+    unless ExerciseStore.isLoaded(pageIds)
+      ExerciseActions.load(courseId, pageIds)
+      return <span className="-loading">Loading...</span>
+
+    false
+
+
+ReviewExercises = React.createClass
+  displayName: 'ReviewExercises'
+
+  propTypes:
+    planId: React.PropTypes.any.isRequired
+    courseId: React.PropTypes.any.isRequired
+    pageIds: React.PropTypes.array
+
+  mixins: [ExercisesRenderMixin]
+
+  renderExercise: (exercise, i) ->
+    <ReviewExerciseCard index={i} planId={@props.planId} exercise={exercise}/>
+
+  render: ->
+    load = @renderLoading()
+    if (load)
+      return load
+    
+    {courseId, pageIds, planId} = @props
+
+    if not TaskPlanStore.getTopics(planId).length
+      return <div className='bug'>Failed loading exercises</div>
```

Comment:
Could you add this class into `tutor.less` and make it big and [blinking](https://stackoverflow.com/questions/16344354/how-to-make-blinking-flashing-text-with-css3/16344548#16344548) (or marquee)?